frictionless/package.py

```diff
@@ -330,38 +330,73 @@ def created(self):
         """
         return self.get("created", "")
 
```

Comment:
Remove `__setattr__` method as it's not implemented using python property setters?